libgalois/include/katana/PropertyGraph.h

```diff
@@ -126,7 +195,8 @@ class KATANA_EXPORT PropertyGraph {
   using Edge = GraphTopology::Edge;
 
 private:
-  PropertyGraph(std::unique_ptr<tsuba::RDGFile> rdg_file, tsuba::RDG&& rdg);
+  PropertyGraph(std::unique_ptr<GraphTopology> topoToAssume) noexcept
```

Comment:
What does "assume" mean here?

Reply:
nit: I'd prefer `std::move` to assuming ownership of `unique_ptr`.

Reply:
assume means assume ownership. I'll change to topo_to_assign if that sounds clearer.

Reply:
Why not use a move reference `unique_ptr&&`? Then there is no need to specify the ownership change in the name since it's explicit in the type.

Reply:
Good suggestion. I've removed the constructor for now but will bring it back soon in my next PR.